src/page-objects/administration/FlowBuilderDetail.ts

```diff
@@ -0,0 +1,19 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class FlowBuilderDetail implements PageObject {
+
+    public readonly saveButton: Locator;
+    public readonly generalTab: Locator;
+    public readonly flowTab: Locator;
+
+    constructor(public readonly page: Page) {
+        this.saveButton = page.locator('.sw-flow-detail__save');
+        this.generalTab = page.locator('.sw-flow-detail__tab-general');
+        this.flowTab = page.locator('.sw-flow-detail__tab-flow');
+    }
+
+    url(flowId: string, tabName? = 'general') {
```

Comment:
```suggestion
    url(flowId: string, tabName: string = 'general') {
```

Reply:
It would be nice if we could make the tab optional to visit any tab without a second click.

Reply:
If a variable has a default value, the type is inferred from that.

Reply:
@yusufttur The tab is already optional with my implementation. If you don't pass it, it will route to the general tab, which is the default.